Tests/StreamChatUITests/SnapshotTests/CommonViews/ChatChannelHeaderView/ChatChannelHeaderView_Tests.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatChannelHeaderView_Tests: XCTestCase {
+    var view: ChatChannelHeaderView!
+    var mockChannelController: ChatChannelController_Mock!
+    
+    override func setUp() {
+        super.setUp()
+        
+        mockChannelController = ChatChannelController_Mock.mock()
+        view = ChatChannelHeaderView().withoutAutoresizingMaskConstraints
+        view.backgroundColor = .gray
+        view.addSizeConstraints()
+        view.channelController = mockChannelController
+    }
+    
+    override func tearDown() {
+        view = nil
+        mockChannelController = nil
+        
+        super.tearDown()
+    }
+    
+    func test_SetNewChannelName_IsSuccessfullyChanged() {
+        // Given
+        let newChannelName = "New Channel Name"
+        let mockChannel: ChatChannel = .mockDMChannel(name: newChannelName)
+        mockChannelController.channel_mock = mockChannel
+        
+        // When
+        view.channelController(view.channelController!, didUpdateChannel: .update(mockChannel))
+        
+        // Then
+        let currentChannelName = view.titleContainerView.titleLabel.text ?? ""
+        XCTAssertEqual(currentChannelName.isEmpty, false)
+        XCTAssertEqual(newChannelName, currentChannelName)
+    }
+    
+    func test_ChannelNameSet() {
+        let newChannelName = "New Channel Name"
+        mockChannelController.simulateInitial(
+            channel: .mockDMChannel(name: newChannelName),
+            messages: [],
+            state: .localDataFetched
+        )
+        
+        AssertSnapshot(view)
+    }
```

Comment:
It is possible to add mocked messages in this case? To test the subtitle of the header view? Otherwise, this test is not required, because it is already covered by the first one 👍

Reply:
Adjusted to also show and test the subtitle. You can see it on the new snapshots added to this PR.